cmd/snap/cmd_model.go

```diff
@@ -46,6 +47,7 @@ model assertion.
 `)
 
 	invalidTypeMessage = i18n.G("invalid type for %q header")
+	devNotReadyMessage = i18n.G("device not ready yet (no assertions found)")
```

Comment:
this can likely be an error directly?  it doesn't have placehoders and we always use it as error text below?

Reply:
yes this error message (and the other one I needed to add) are both errors directly